2012-05-16 13:13:54.499 [error] <0.118.0> CRASH REPORT Process <0.118.0> with 0 neighbours crashed with reason: call to undefined function lager:info("Starting HTTP server on ~p", [3000])
2012-05-16 13:13:54.524 [error] <0.117.0> Supervisor hello_world_sup had child hello_world_vnode_dispatcher_master started with hello_world_vnode_dispatcher:start_link() at undefined exit with reason call to undefined function lager:info("Starting HTTP server on ~p", [3000]) in context start_error
2012-05-16 13:19:09.430 [error] <0.119.0> CRASH REPORT Process <0.119.0> with 0 neighbours crashed with reason: call to undefined function lager:info("Starting HTTP server on ~p", [3000])
2012-05-16 13:19:09.434 [error] <0.118.0> Supervisor hello_world_sup had child hello_world_vnode_dispatcher_master started with hello_world_vnode_dispatcher:start_link() at undefined exit with reason call to undefined function lager:info("Starting HTTP server on ~p", [3000]) in context start_error
2012-05-16 14:57:50.742 [error] <0.119.0> CRASH REPORT Process <0.119.0> with 0 neighbours crashed with reason: call to undefined function lager:info("Starting HTTP server on ~p", [3000])
2012-05-16 14:57:50.746 [error] <0.118.0> Supervisor hello_world_sup had child hello_world_vnode_dispatcher_master started with hello_world_vnode_dispatcher:start_link() at undefined exit with reason call to undefined function lager:info("Starting HTTP server on ~p", [3000]) in context start_error
2012-05-16 15:00:37.539 [error] <0.119.0> CRASH REPORT Process <0.119.0> with 0 neighbours crashed with reason: call to undefined function lager:info("Starting HTTP server on ~p", [3000])
2012-05-16 15:00:37.543 [error] <0.118.0> Supervisor hello_world_sup had child hello_world_vnode_dispatcher_master started with hello_world_vnode_dispatcher:start_link() at undefined exit with reason call to undefined function lager:info("Starting HTTP server on ~p", [3000]) in context start_error
2012-05-16 15:00:51.135 [error] <0.202.0> CRASH REPORT Process <0.202.0> with 0 neighbours crashed with reason: call to undefined function lager:debug("started ~p at partition ~p", [hello_world_vnode,0])
2012-05-16 15:00:51.141 [error] <0.96.0> Supervisor riak_core_vnode_sup had child undefined started with {riak_core_vnode,start_link,undefined} at <0.202.0> exit with reason call to undefined function lager:debug("started ~p at partition ~p", [hello_world_vnode,0]) in context child_terminated
2012-05-16 15:00:51.145 [error] <0.111.0> gen_server riak_core_vnode_manager terminated with reason: no match of right hand value {error,{undef,[{lager,debug,["started ~p at partition ~p",[hello_world_vnode,0]]},{hello_world_vnode,init,1},{riak_core_vnode,init,1},{gen_fsm,init_it,6},{proc_lib,init_p_do_apply,3}]}} in riak_core_vnode_manager:get_vnode/3
2012-05-16 15:00:51.152 [error] <0.111.0> CRASH REPORT Process riak_core_vnode_manager with 0 neighbours crashed with reason: no match of right hand value {error,{undef,[{lager,debug,["started ~p at partition ~p",[hello_world_vnode,0]]},{hello_world_vnode,init,1},{riak_core_vnode,init,1},{gen_fsm,init_it,6},{proc_lib,init_p_do_apply,3}]}} in riak_core_vnode_manager:get_vnode/3
2012-05-16 15:00:51.156 [error] <0.93.0> Supervisor riak_core_sup had child riak_core_vnode_manager started with riak_core_vnode_manager:start_link() at <0.111.0> exit with reason no match of right hand value {error,{undef,[{lager,debug,["started ~p at partition ~p",[hello_world_vnode,0]]},{hello_world_vnode,init,1},{riak_core_vnode,init,1},{gen_fsm,init_it,6},{proc_lib,init_p_do_apply,3}]}} in riak_core_vnode_manager:get_vnode/3 in context child_terminated
2012-05-16 15:04:53.764 [error] <0.131.0> CRASH REPORT Process <0.131.0> with 0 neighbours crashed with reason: call to undefined function lager:debug("started ~p at partition ~p", [hello_world_vnode,0])
2012-05-16 15:04:53.774 [error] <0.95.0> Supervisor riak_core_vnode_sup had child undefined started with {riak_core_vnode,start_link,undefined} at <0.131.0> exit with reason call to undefined function lager:debug("started ~p at partition ~p", [hello_world_vnode,0]) in context child_terminated
2012-05-16 15:04:53.795 [error] <0.128.0> gen_server hello_world_vnode_master terminated with reason: no match of right hand value {error,{undef,[{lager,debug,["started ~p at partition ~p",[hello_world_vnode,0]]},{hello_world_vnode,init,1},{riak_core_vnode,init,1},{gen_fsm,init_it,6},{proc_lib,init_p_do_apply,3}]}} in riak_core_vnode_master:get_vnode/2
2012-05-16 15:04:53.805 [error] <0.128.0> CRASH REPORT Process hello_world_vnode_master with 0 neighbours crashed with reason: no match of right hand value {error,{undef,[{lager,debug,["started ~p at partition ~p",[hello_world_vnode,0]]},{hello_world_vnode,init,1},{riak_core_vnode,init,1},{gen_fsm,init_it,6},{proc_lib,init_p_do_apply,3}]}} in riak_core_vnode_master:get_vnode/2
2012-05-16 15:04:53.811 [error] <0.112.0> Supervisor hello_world_sup had child hello_world_vnode_master started with riak_core_vnode_master:start_link(hello_world_vnode) at <0.128.0> exit with reason no match of right hand value {error,{undef,[{lager,debug,["started ~p at partition ~p",[hello_world_vnode,0]]},{hello_world_vnode,init,1},{riak_core_vnode,init,1},{gen_fsm,init_it,6},{proc_lib,init_p_do_apply,3}]}} in riak_core_vnode_master:get_vnode/2 in context child_terminated
2012-05-16 15:07:12.521 [error] <0.248.0> 	module: misultin_http
	line: 708
error in custom loop: {undef,[{lager,debug,["Dispatching to ~p",[{1118962191081472546749696200048404186924073353216,'helloworld1@localhost'}]]},{hello_world_vnode_dispatcher,dispatch,1}]} serving request: {req,#Port<0.6179>,http,{127,0,0,1},49708,undefined,keep_alive,undefined,{1,1},'GET',{abs_path,"/"},[],[{'User-Agent',"curl/7.21.6 (x86_64-pc-linux-gnu) libcurl/7.21.6 OpenSSL/1.0.0e zlib/1.2.3.4 libidn/1.22 librtmp/2.3"},{'Host',"localhost:3000"},{'Accept',"*/*"}],<<>>,false}
2012-05-16 15:07:12.594 [error] emulator Error in process <0.249.0> on node 'helloworld1@localhost' with exit value: {undef,[{lager,debug,["Dispatching to ~p",[{1118962191081472546749696200048404186924073353216,'helloworld1@localhost'}]]},{hello_world_vnode_dispatcher,dispatch,1}]}

2012-05-16 15:15:05.551 [error] <0.98.0> CRASH REPORT Process riak_core_handoff_listener with 1 neighbours crashed with reason: bad return value: {error,eaddrinuse}
2012-05-16 15:15:05.560 [error] <0.92.0> Supervisor riak_core_sup had child riak_core_handoff_listener started with riak_core_handoff_listener:start_link() at undefined exit with reason bad return value: {error,eaddrinuse} in context start_error
2012-05-16 15:15:19.494 [error] <0.98.0> CRASH REPORT Process riak_core_handoff_listener with 1 neighbours crashed with reason: bad return value: {error,eaddrinuse}
2012-05-16 15:15:19.501 [error] <0.92.0> Supervisor riak_core_sup had child riak_core_handoff_listener started with riak_core_handoff_listener:start_link() at undefined exit with reason bad return value: {error,eaddrinuse} in context start_error
